resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/request/AddResourceRequest.java

```diff
@@ -26,37 +29,34 @@
 import static java.util.Objects.requireNonNull;
 
 /**
- * This class is used to send a request to the
- * DataService to read a resource.
+ * This class is used to request that details about a resource is added to the resource-service.
  */
-
-public class ReadRequest extends Request {
-    private String token;
+@JsonIgnoreProperties(value = {"originalRequestId"})
+public class AddResourceRequest extends Request {
```

Comment:
Why is the ReadRequest being changed to an AddResourceRequest? Is this a git thing as well?

Reply:
Git thing, its because the copyright takes up such a large chunk of the file, and that bit obviously didnt change, so they're _totally_ the same file apparently...